dashboard/src/components/jobs/parameters/DataParams.vue

```diff
@@ -0,0 +1,292 @@
+          effective irradiance.
+        </label>
+        <br />
+      </div>
+    </div>
+    <div class="my-1">
+      How should we determine cell temperature?
+      <br />
+      <div class="ml-1 mt-1">
+        <label>
+          <input
+            @change="emitParams"
+            value="air"
+            type="radio"
+            v-model="temperature_type"
+          />
+          Calculate cell temperature from irradiance, air temperature, and
+          windspeed in my data.
+        </label>
+        <br />
+        <label>
+          <input
+            @change="emitParams"
+            value="module"
+            type="radio"
+            v-model="temperature_type"
+          />
+          Calculate cell temperature from module temperature and irradiance in
+          my data.
+        </label>
+        <br />
+        <label>
+          <input
+            @change="emitParams"
+            value="cell"
+            type="radio"
+            v-model="temperature_type"
+          />
+          Cell temperature is included in my data.
+        </label>
+        <br />
+      </div>
+    </div>
+    <div class="my-1">
+      I will provide weather data as:
+      <br />
+      <div class="ml-1 mt-1">
+        <label>
+          <input
+            @change="emitParams"
+            value="system"
+            type="radio"
+            v-model="weather_granularity"
+          />
+          one set for the entire system.
+        </label>
+        <br />
+
+        <label>
+          <input
+            @change="emitParams"
+            value="inverter"
+            type="radio"
+            v-model="weather_granularity"
+          />
+          one set for each inverter and its associated arrays.
+        </label>
+        <br />
+        <label>
+          <input
+            @change="emitParams"
+            value="array"
+            type="radio"
+            v-model="weather_granularity"
+          />
+          one set for each array.
+        </label>
+        <br />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from "vue-property-decorator";
+
+@Component
+export default class DataParams extends Vue {
+  @Prop() dataType!: string;
+  @Prop() jobClass!: string;
+  data_available!: string;
+  weather_granularity!: string;
+  irradiance_type!: string;
+  temperature_type!: string;
+  performance_granularity!: string;
+
+  data() {
+    return {
+      data_available: "weather only",
+      irradiance_type: "standard",
+      weather_granularity: "system",
+      temperature_type: "air",
+      performance_granularity: "system"
+    };
+  }
+  created() {
+    if (this.jobClass == "compare") {
+      // Ensure the correct data availability is set for compare usecase
+      // This field will not be included in the posted job JSON for
+      // dataTypes other than predicted.
+      if (this.dataType == "predicted") {
+        this.data_available = "weather only";
```

Comment:
can we make 'weather and AC performance' the default? Good to keep the same order, but we should direct people to that one probably

Reply:
for both predicted and actual, good to set irradiance to poa and temp to cell